### PR TITLE
162238098 create landing page

### PR DESCRIPTION
**Pull request by MuhanguziDavid**
### What does this PR do?
Adds a landing page to be displayed on navigating to the main URL.

#### Description of Task to be completed?
Currently, there is no landing page that gives users options of signing up or logging in.
This PR adds that feature

#### How should this be manually tested?
Run `npm install`
Run `npm start`.
Access `http://localhost:3000` and the page will be rendered.

#### Background context?
N/A

#### What are the relevant pivotal tracker stories?
[162238098](https://www.pivotaltracker.com/story/show/162238098)

#### Screenshots
screen shot 2018-11-28 at 12 48 25

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2423,6 +2423,13 @@
         "base64-js": "^1.0.2",
```
Comment (mendozabree) [inline package-lock.json]: @MuhanguziDavid this file should be added to your git.ignore

Comment (MuhanguziDavid) [reply package-lock.json]: Thank you @mendozabree. `package-lock.json` has been added to `.gitignore`

File: public/index.html [hunk 2]
```diff
@@ -3,8 +3,12 @@
   <head>
     <meta charset="utf-8">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css"
```
Comment (samuelbwambale) [inline public/index.html]: There are two Bootstrap CDN links. Please consider choosing one of them

Comment (MuhanguziDavid) [reply public/index.html]: Thanks, @samuelbwambale, this has been dealt with accordingly.

**Review by samuelbwambale — APPROVED** [review]
LGTM

**Pull request merged**